Makefile

```diff
@@ -1,2 +1,5 @@
 test:
 	./python_tests.sh
+
+avro:
+	PYTHONPATH=$(PYTHONPATH):./py/ python scripts/generate_avro.py
```

Comment:
If you mean `python3`, then you should say that. At least I don't have my machine configured the Instabase Way™ and I don't think Dominique does either.